CoffeeShop.playground/Sources/CoffeeShopTest.swift

```diff
@@ -0,0 +1,61 @@
+import Foundation
+import XCTest
+
+extension CoffeeShop: Equatable {
```

Comment:
@eandreea As a good practice, is not necessary to extend existing functionality just to write tests. For example, instead of comparing coffeeShop!.location with  Location(x: 47.5809, y: -122.3160), you can compare  coffeeShop!.location.x with 47.5809

Reply:
I agree, but in a previous comment it was stated that this functionality was already tested in the Location test, so it's just a duplicate. But to test that the CoffeeShop instance was correctly created I have to test the location parameter too... In the meantime i've seen that, after refactoring, only Location needs to conform to Equatable, so I'll remove the one from CoffeeShop

Reply:
Great. Sorry for contradictory comments, we didn't sync the comments across the team. As long this is the result of another comment than is ok for me.